assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -125,13 +133,7 @@ function Header({
         handleLayoutSelect={view.toggleStyle}
         currentSort={sort.value}
         pageSortOptions={STORY_SORT_MENU_ITEMS}
-        handleSortChange={useCallback(
-          (newSort) => {
-            sort.set(newSort);
-            scrollToTop();
-          },
-          [scrollToTop, sort]
-        )}
+        handleSortChange={onSortChange}
```

Comment:
this is just a nit that pascal asked for yesterday in the rush for beta, making good on the update :D